src/synapse/azext_synapse/_validators.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def example_name_or_id_validator(cmd, namespace):

```

Comment:
This example can be removed. Please add any necessary arguments check or processing here.

Reply:
Can we add it in next version?

---
In reply to: [387016467](https://github.com/Azure/azure-cli-extensions/pull/1339#discussion_r387016467) [](ancestors = 387016467)

Reply:
sure.

Reply:
Add TODO here to track future work item.

---
In reply to: [388148347](https://github.com/Azure/azure-cli-extensions/pull/1339#discussion_r388148347) [](ancestors = 388148347)